source/optional_features/custom_error_pages/index.html.md.erb

```diff
@@ -0,0 +1,24 @@
+---
+title: Custom error pages 
+weight: 135
+---
+
+# Custom error pages
+
+GOV.UK Pay supports custom error pages.
+
+When a payment reaches a terminal, unsuccessful state, GOV.UK Pay will
+immediately redirect the user to the return URL (`return_url`), which you can
+set in the response to the `POST /v1/payments` API call. 
+
+To set this up for your service, you should make a query to the GOV.UK Pay API
```

Comment:
this doesn't quite describe how a service would set it up - they need to contact us for the feature to be enabled. also, regardless of whether this feature is used, a service would still make a GET request per transaction and (ideally) should parse the failure state so they can direct the user on the right journey. For example, if the user cancelled the payment, the service might determine that the user shouldn't be taken back to the start of a payment journey again, whereas if the payment expired, they might.